### DISCUSSION TITLE
TensorFlow Lite Plugin integration

### DISCUSSION THREAD

**Discussion by ResetPress** [flet-dev/flet#4901] [Extensions]
Google recently released a TensorFlow Lite plugin for Flutter:
https://blog.tensorflow.org/2023/08/the-tensorflow-lite-plugin-for-flutter-officially-available.html

Github Repo:
https://github.com/tensorflow/flutter-tflite

Integration with Flet would be useful for anyone interested in running machine learning models locally (mobile included) or via the web.

**Comment by ndonkoHenri** [comment]
https://github.com/flet-dev/flet/discussions/3810#discussioncomment-17672290
